projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css

```diff
@@ -47,6 +47,49 @@ button.red:hover, button.red:focus {
   background-color: #ff0000;
 }
 
+button.dropdown-toggle {
+  padding-left: 3px;
+  padding-right: 6px;
+  border-left: 1px solid transparent;
+  margin-left: 1px;
+}
+
+/* Only visible to screen reader */
+button.dropdown-toggle span {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  white-space: nowrap;
+  -webkit-clip-path: inset(50%);
+  clip-path: inset(50%);
+  border: 0;
+}
+
+button.dropdown-toggle::after {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin-left: .255em;
+  vertical-align: .255em;
+  content: "";
+  border-top: .3em solid;
+  border-right: .3em solid transparent;
+  border-bottom: 0;
+  border-left: .3em solid transparent
```

Comment:
Is this related to the icon placement?

Reply:
This creates the triangle icon (it's just CSS border properties, no SVG or anything). I swiped and modified a lot of this from Bootstrap.

Reply:
And to answer your other question (since you inexplicably can't reply to top-level comments in PRs) this is indeed a pre-req for a Publish feature! Specifically prx/publish.prx.org#579

Reply:
Ahh right! `border-bottom: 0;` I assumed this was related to @rpeterman-gp work on svg/icon abstractions. Would it makes sense to use that here?

Reply:
That would be an equally reasonable approach to this problem. Either way, it’s just a display element and there’s hidden text for accessibility.